pkg/infrastructure/clusterapi/helpers.go

```diff
@@ -32,3 +34,22 @@ func injectInstallInfo(bootstrap []byte) ([]byte, error) {
 
 	return ign, nil
 }
+
+func prioritizeIPv4(config *types.InstallConfig, addresses []string) string {
+	if len(addresses) > 1 {
+		var ipv4addr string
+		for _, a := range addresses {
+			if !strings.Contains(a, ":") {
```

Comment:
Should we use https://pkg.go.dev/net#ParseIP or do you think it's overkill?

Reply:
Oh I see it should be enough if we expect the address to contain only the host part: https://github.com/asaskevich/govalidator/blob/3b2665001c4c24e3b076d1ca8c428049ecbb925b/validator.go#L501

Reply:
I like using ParseIP better, updated, thx!